### PR TITLE
Merge Platform.Bible in extension template

### PR DESCRIPTION
**Pull request by tombogle**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/119)

### CODE REVIEW

**Review by tjcouch-sil — APPROVED** [review]
:lgtm: though I'd still recommend against this because, in my opinion that doesn't matter, it clutters your repo and makes it hard to keep track of what is what. Though I am a bit concerned - can you still merge updates from the template like this? That's the only real concern I have.

> Reviewed 26 of 26 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/119#-:-Nd1ZH1C8Pc4NXIVgLYh:b-63x6u7)* status: all files reviewed, 1 unresolved discussion (waiting on @katherinejensen00 and @tombogle)

___
*[`LICENSE` line 3 at r1](https://reviewable.io/reviews/sillsdev/Transcelerator/119#-Nd1YBOF-O_TLc99aRi2:-Nd1YBOF-O_TLc99aRi3:b-v3qpyz) ([raw file](https://github.com/sillsdev/Transcelerator/blob/144412cd6e9206308068d33bbb2663de9270133e/LICENSE#L3)):*
> ```
> MIT License
> 
> Copyright (c) 2023 Paranext
> ```

Is this license correct?

**Comment by tombogle** [comment]
Since initially merging in the template, I have twice merged in updates, so there doesn't appear to be any problem with that. I don't guess it's any messier than many of the other repos that are used to produce multiple artifacts.

**Comment by tombogle** [comment]
*[`LICENSE` line 3 at r1](https://reviewable.io/reviews/sillsdev/Transcelerator/119#-Nd1YBOF-O_TLc99aRi2:-Nd1fAXc4R0PcGNdqK_R:byaousw) ([raw file](https://github.com/sillsdev/Transcelerator/blob/144412cd6e9206308068d33bbb2663de9270133e/LICENSE#L3)):*
<details><summary><i>Previously, tjcouch-sil (TJ Couch) wrote…</i></summary><blockquote>

Is this license correct?
</blockquote></details>

It is now

File: src/main.ts [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+  logger,
+  dataProvider: { DataProviderEngine },
+} = papi;
+
+console.log(process.env.NODE_ENV);
+
+logger.info('Transcelerator is importing!');
+
+/**
+ * Example data provider engine that provides easy access to Scripture from an internet API.
```
Comment (katherinejensen00) [inline src/main.ts:33]: You probably don't need this comment, but it doesn't hurt.

**Review by katherinejensen00 — COMMENTED** [review]
Sorry this review is late. This looks good. Thanks for the good work, Tom!

**Pull request merged**